frontend/public/components/_icon-and-text.scss

```diff
@@ -1,6 +1,6 @@
 .co-icon-and-text {
   align-items: baseline;
-  display: flex;
+  display: inline;
```

Comment:
This is going to affect all places this class is used. It was certainly `flex` for a reason. We can't apply this change globally without understanding the impact to other places.

Can you use `ResourceLink` instead with the `inline` prop set? That way we get to badge to be consistent with other resource links and don't repeat the URL building logic.

Reply:
it won't work because we are giving "StatusIcon" tag before Resource link in the same column.  StatusIcon tag occupies hole row. 

As per the new commit, I have sperate those values into two different column